root/genre/DeleteGenre.js

```diff
@@ -28,12 +28,12 @@ const DeleteGenre = ({
     entity={genre}
     fullWidth
     page="delete"
-    title={lp('Remove genre', 'header')}
+    title={'Remove genre'}
```

Comment:
Thought we had an ESLint rule for this, but a normal string would typically just be passed like:
```suggestion
    title="Remove genre"
```

Reply:
I actually did that for some but missed the titles. Amending.